static_test_driver/static_test_driver.ino

```diff
@@ -87,16 +87,14 @@ int accel_error = 0;
 bool sensor_status = true;
 
 // Engine control setup
-typedef enum {
```

Comment:
I fixed the bug with the buttons doing different things (it was a bug in how I layed out buttons in the GUI), so everything related to the valve control should be changed back.